pkgs/development/python-modules/ruamel-yaml-string/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "ruamel-yaml-string";
+  version = "0.1.1";
```

Comment:
```suggestion
  version = "0.1.1";
  pyproject = true;
```

Reply:
The project doesn't have a pyproject file. Setting this to `true` causes a build failure.